src/js/steps/generateBaseWrapperElement.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var $ = require('jquery');
+
+/**
+ * Generate base HTML element for all of monkey.
+ *
+ * Varies depending on the browser.
+ *
+ */
+module.exports = function ($monkeyContainer) {
+  return function (data) {
```

Comment:
Could be more DRY i.e. put the bits that change in the conditional rather than the whole block

```js
var classes = data.monkeyType === mobile ? 'monkey-wrapper mobile' : 'pos-relative positioned-relative monkey-wrapper desktop';
data.$monkeyWrapper.addClass(classes).appendTo($monkeyContainer);
return data;
```